cpp/src/barretenberg/proof_system/op_queue/ecc_op_queue.hpp

```diff
@@ -85,8 +111,18 @@ class ECCOpQueue {
             .mul_scalar_full = scalar,
         });
     }
-    void eq(const Point& expected)
+
+    /**
+     * @brief Write equality op to queue and check equality with native accumulator
+     *
+     * @param expected
+     * @return whether accumulator point == expected point
+     */
+    bool eq(const Point& expected)
```

Comment:
This interface forces us to always redo this computation. First you need to compute the value externally, then the same value is computed in OpQueue. What do you think of making the return value the point and having no argument to the function? Then we can check the value correctness in tests

Reply:
Changed this so that instead of calling `get_accumulator()` explicitly, the equality op implicitly uses used the internal accumulator and that value is returned when calling `queue_ecc_eq()` for use elsewhere in the circuit if necessary.